docs/tools/cli/service/privatelink.rst

```diff
@@ -0,0 +1,423 @@
+``avn service privatelink``
+==============================================
+
+Here you’ll find the full list of commands for ``avn service privatelink``.
+
+
+Manage Aiven PrivateLink service for AWS and Azure
+--------------------------------------------------------
+
+.. _avn_service_privatelink_availability:
+
+``avn service privatelink availability``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Lists PrivateLink cloud availability and prices.
```

Comment:
I'll add parameters. However, I was hesitant to add an example with output since it's showing pricing, which can change over time, and the pricing list is pretty long. Let me know if that's fine.

Reply:
We can always cut a bit of the response, we don't need to show it all